packages/calcite-components/src/components/navigation/navigation.scss

```diff
@@ -3,9 +3,13 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-navigation-width: Specifies the width of the component's content area.
+ * @prop --calcite-navigation-action-background-color: defines the background color of an action sub-component inside the component.
+ * @prop --calcite-navigation-action-indicator-color: defines the indicator color of an action sub-component inside the component.
```

Comment:
Similar to others, this can't be set.
We can addd hover and active pseudo states to this action because it is internally rendered and a user can't override those on parent component state.